libraries/AP_HAL_Linux/Util.cpp

```diff
@@ -91,16 +91,12 @@ void Util::_toneAlarm_timer_tick() {
 
 }
 
-void Util::set_system_clock(uint64_t time_utc_usec)
+void Util::set_hw_rtc(uint64_t time_utc_usec)
 {
 #if CONFIG_HAL_BOARD_SUBTYPE != HAL_BOARD_SUBTYPE_LINUX_NONE
-    _system_time_was_set = true;
-
-    timespec ts;
-    ts.tv_sec = time_utc_usec/1000000ULL;
-    ts.tv_nsec = (time_utc_usec % 1000000ULL) * 1000ULL;
-    clock_settime(CLOCK_REALTIME, &ts);
+    return;
```

Comment:
This seems totally wrong, it means all Linux boards except Linux_None won't set the time on hardware. I also don't see why Linux_None wouldn't set the time, it's still Linux, just no specific board.